app/helpers/administration/settings_helper.rb

```diff
@@ -48,4 +48,8 @@ def setting_description(setting)
     end
   end
 
+  def zoom_options
+    (12..16).map{|zoom_level| [I18n.t('name', scope: ['admin.settings', 'System', "Zoom Level #{zoom_level}"]), zoom_level]}
```

Comment:
Can you put spaces around your curly braces and wrap this to multiple lines since it's over 120?

We try to follow [bbatsov style guide](https://github.com/bbatsov/ruby-style-guide) best we can.